connectors/jira/src/main/java/com/vmware/connectors/jira/JiraController.java

```diff
@@ -230,7 +231,8 @@ private void addCommentsField(CardBody.Builder cardBodyBuilder, List<String> all
                     .setType(CardBodyFieldType.COMMENT);
 
             cardFieldBuilder.addContent(ImmutableMap.of("text", allComments.get(0)));
-            if (allComments.size() > 1) {
+            int commentsSize = 1;
+            if (allComments.size() > commentsSize) {
```

Comment:
This fixes another "controversial" rule violation and makes the code less readable.

Reply:
Created a class variable for comments size and referring it to the if condition.